packages/application-shell-connectors/src/components/project-extension-image-regex/project-extension-image-regex.tsx

```diff
@@ -61,16 +61,10 @@ const GetProjectExtensionImageRegex = (props: ConsumerProps) => (
 );
 GetProjectExtensionImageRegex.displayName = 'GetProjectExtensionImageRegex';
 
-function withProjectExtensionImageRegex<Props extends {}>(
-  propKey = 'imageRegexData'
-) {
+function withProjectExtensionImageRegex<Props extends {}>() {
```

Comment:
Same thing here, I guess this HOC should be left as it was. Otherwise it will be a breaking change.